blank_react_project/src/App.js

```diff
@@ -16,6 +16,7 @@ class App extends Component {
     this.requestSignIn = this.requestSignIn.bind(this);
     this.requestSignOut = this.requestSignOut.bind(this);
     this.signedOutFlow = this.signedOutFlow.bind(this);
+    this.changeGreeting = this.changeGreeting.bind(this);
```

Comment:
why do we need these? That doesn't look right, we don't do such stuff in NEAR Wallet.

Reply:
this is null inside changeGreeting unless this line is present.